windpowerlib/wind_turbine.py

```diff
@@ -163,6 +163,11 @@ def __init__(self, hub_height, nominal_power=None, path='oedb',
                 if self.rotor_diameter is None and turbine_data is not None:
                     self.rotor_diameter = float(turbine_data['rotor_diameter'])
 
+        if self.rotor_diameter:
+            if self.hub_height <= 0.5*self.rotor_diameter:
+                msg = "1/2*rotor_diameter cannot be higher than hub_height"
+                raise ValueError(msg)
```

Comment:
I am not a native speaker but as I know it in a mathematical sense one value should not be GREATER than another not HIGHER.

Reply:
Changed.